mne/viz/_3d.py

```diff
@@ -345,6 +352,31 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
         subjects_dir = get_subjects_dir(subjects_dir, raise_error=True)
         surfs['head'] = get_head_surf(subject, source=source,
                                       subjects_dir=subjects_dir)
+    if fiducials:
+        from ..coreg import _fiducial_coords  # avoid circular import
+
+        if fiducials is True:
+            subjects_dir = get_subjects_dir(subjects_dir, raise_error=True)
+            if subject is None:
+                raise ValueError("Subject needs to be specified to "
+                                 "automatically find the fiducials file.")
+            fiducials = op.join(subjects_dir, subject, 'bem',
+                                subject + '-fiducials.fif')
+        if isinstance(fiducials, string_types):
+            fiducials, cf = read_fiducials(fiducials)
+            assert cf == FIFF.FIFFV_COORD_MRI
+        fid_locs = _fiducial_coords(fiducials, FIFF.FIFFV_COORD_MRI)
+        if coord_frame == 'head':
+            mri_head_t = invert_transform(head_mri_t)
+            fid_locs = apply_trans(mri_head_t, fid_locs)
+        elif coord_frame == 'meg':
+            mri_head_t = invert_transform(head_mri_t)
+            head_dev_t = invert_transform(info['dev_head_t'])
+            mri_dev_t = combine_transforms(mri_head_t, head_dev_t, 'mri',
+                                           'meg')
+            fid_locs = apply_trans(mri_dev_t, fid_locs)
```

Comment:
https://github.com/mne-tools/mne-python/pull/3999/commits/8abb42b6e31a268c080abfdc702d3188a71e7d4b of https://github.com/mne-tools/mne-python/pull/3999